### DISCUSSION TITLE
[@types/react] nonce is required now when it used to not be

### DISCUSSION THREAD

**Discussion by DrewLandgraveCbsi** [DefinitelyTyped/DefinitelyTyped#64409] [Issues with a @types package]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62879 Added the nonce attribute to the global scope. I think this may be causing issues with other components. I have a component library that i'm using webpack module federation to pull into another app. It's now telling me 

```
Type '{ required: true; label: string; defaultValue: string; disabled: true; variant: "filled"; margin: "normal"; }' is not assignable to type 'IntrinsicAttributes & ((Pick<TextFieldProps & StandardTextFieldProps, "label" | "slot" | "select" | "style" | ... 282 more ... | keyof TextFieldProps> | Pick<...> | Pick<...>) & RefAttributes<...>)'.
[0]   Property 'nonce' is missing in type '{ required: true; label: string; defaultValue: string; disabled: true; variant: "filled"; margin: "normal"; }' but required in type 'Pick<TextFieldProps & FilledTextFieldProps, "label" | "slot" | "select" | "style" | "title" | "children" | "placeholder" | "className" | ... 278 more ... | keyof TextFieldProps>'.
```

It's coming from the Material-ui/TextField but the nonce is coming from @types/react. 

I'm not sure what version it was added in but I used to be using 17.0.39 and recently upgraded to 17.0.53

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment]
So utility type is probably misconfigured. Or you maybe don't have `strictNullCheckes`? Either way, without a minimal, cloneable repro, we can't help you since the `nonce` is clearly marked as optional in the linked PR.
